generators/environment/templates/outrigger/outrigger.yml

```diff
@@ -19,53 +19,66 @@ scripts:
     alias: run
     description: Run one of the build container services such as 'cli' or 'drush'.
     run:
-      - docker-compose -f $PWD/build.yml run --rm
-
-  setup:
-    alias: setup
-    description: Run the end-to-end repository initialization and site install script.
-    run:
-      - rig project run:welcome
-      - rig project sync:start
-      - start.sh
+      - if [ "$OSTYPE" == "linux-gnu" ]; then COMPOSE_FILE=build.devcloud.yml; else COMPOSE_FILE=build.yml; fi
```

Comment:
I'm not sure I like this convention. I may rather a key when we want to support single or mutliple OS types. Something like...

```
setup:
  alias: setup
  description: Run the end-to-end repository initialization and site install script.
  run:
  - rig project run:welcome
  - rig project sync:start
  - start.sh
  - linux-gnu:  COMPOSE_FILE=build.devcloud.yml
    default: COMPOSE_FILE=build.yml
  - linux-gnu: one-thing 
    osx: another-thing
    windows: yet-another-thing
```

Reply:
One approach @mike-potter raised was something more like this:

```
setup:
  alias: setup
  description: Run the end-to-end repository initialization and site install script.
  run:
    linux:
      - rig project run:welcome
      - rig project sync:start
      - COMPOSE_FILE=build.devcloud.yml
      - start.sh
```

It seems pretty ungainly to me, which is why I started talking about supporting outrigger.linux.yml and have it merge like docker-compose.overrides.yml

While I'd like to figure out a good way to make rig project configuration more OS-adaptive, I want to get start.sh pulled more. Does this represent a change we can live with for a couple months worth of new projects?